patches/net/minecraft/world/item/alchemy/PotionBrewing.java.patch

```diff
@@ -56,11 +56,22 @@
              Optional<Holder<Potion>> optional = p_43531_.getOrDefault(DataComponents.POTION_CONTENTS, PotionContents.EMPTY).potion();
              if (optional.isEmpty()) {
                  return p_43531_;
-@@ -127,6 +_,7 @@
+@@ -124,9 +_,18 @@
+         }
+     }
+ 
++    /**
++     * @deprecated Use {@link #bootstrap(FeatureFlagSet, net.minecraft.core.RegistryAccess)} instead
++     */
++    @Deprecated(forRemoval = true, since = "1.21")
```

Comment:
The deprecation should be since 1.21.1.
```suggestion
+    @Deprecated(forRemoval = true, since = "1.21.1")
```

Actually, should we even be marking this as `forRemoval`, since this is a vanilla Minecraft method?

Reply:
That is actually a very good point, it's the same as other vanilla methods where we deprecate them in favour of stack sensitive versions or with additional parameters, I'll fix this